pubspec.lock

```diff
@@ -119,7 +119,7 @@ packages:
       name: characters
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.2.0"
+    version: "1.1.0"
```

Comment:
Not sure what's causing the discrepancy in our lock files, but probably best not to downgrade dependencies if not needed. :)